src/main/view/screen/SettingsPanel.java

```diff
@@ -1,32 +1,392 @@
+	            @Override
+	            public void handle(ActionEvent event) {
+	                saveControlsFileChoose();
+	            }
+	        });
+			exitToMenuButton.setTranslateX(705);
+			exitToMenuButton.setTranslateY(7);
+			exitToMenuButton.setId("button");
+			exitToMenuButton.setOnAction(new EventHandler<ActionEvent>() {
+	            @Override
+	            public void handle(ActionEvent event) {
+	            	updateControlDisplay();
+	    			saveFile(new File(PANDA_DAT_LOCATION), pandaControls);
+	    			saveFile(new File(HUMAN_DAT_LOCATION), humanControls);
+	                panelManager.setMode(ViewEnum.MAIN_MENU);
+	            }
+	        });
+	        general.getStyleClass().setAll("buttonSelected");
+			general.setTranslateX(165);
+	        general.setTranslateY(7);
+	        general.setOnAction(new EventHandler<ActionEvent>() {
+	            @Override
+	            public void handle(ActionEvent event) {
+	            	currentMode = SettingsEnum.GENERAL;
+	                general.getStyleClass().setAll("buttonSelected");
+	                humanControlsButton.getStyleClass().setAll("button");
+	                pandaControlsButton.getStyleClass().setAll("button");
+	                humanControlsButton.setSelected(false);
+	                pandaControlsButton.setSelected(false);
+	            }
+	        });
+	        humanControlsButton.getStyleClass().setAll("button");
+	        humanControlsButton.setTranslateX(270);
+	        humanControlsButton.setTranslateY(7);
+	        humanControlsButton.setOnAction(new EventHandler<ActionEvent>() {
+	            @Override
+	            public void handle(ActionEvent event) {
+	            	saveCurrentSelections();
+	            	currentMode = SettingsEnum.HUMAN;
+	            	updateControlDisplay();
+	                humanControlsButton.getStyleClass().setAll("buttonSelected");
+	                general.getStyleClass().setAll("button");
+	                pandaControlsButton.getStyleClass().setAll("button");
+	                general.setSelected(false);
+	                pandaControlsButton.setSelected(false);
+	            }
+	        });
+	        pandaControlsButton.getStyleClass().setAll("button");
+	        pandaControlsButton.setTranslateX(357);
+	        pandaControlsButton.setTranslateY(7);
+	        pandaControlsButton.setOnAction(new EventHandler<ActionEvent>() {
+	            @Override
+	            public void handle(ActionEvent event) {
+	            	saveCurrentSelections();
+	            	currentMode = SettingsEnum.PANDA;
+	            	updateControlDisplay();
+	                pandaControlsButton.getStyleClass().setAll("buttonSelected");
+	                general.getStyleClass().setAll("button");
+	                humanControlsButton.getStyleClass().setAll("button");
+	                general.setSelected(false);
+	                humanControlsButton.setSelected(false);
+	            }
+	        });
+			settings.getChildren().add(loadButton);
```

Comment:
There's a few cases of this in the file. I guess just see if there's any way to fix it. If not we might have to ask him about it to make sure its okay, because TDA is like a super no-no on this iteration.